__tests__/helpers/create-user.js

```diff
@@ -0,0 +1,43 @@
+const { gql } = require('@apollo/client');
```

Comment:
Please rename this file to `users.js` and put your helper functions for user in this file.

Reply:
Done